things/service.go

```diff
@@ -530,6 +530,30 @@ func (svc service) ListClientsByGroup(ctx context.Context, token, groupID string
 	}, nil
 }
 
+func (svc service) SearchThings(ctx context.Context, token string, pm mgclients.Page) (mgclients.ClientsPage, error) {
+	res, err := svc.identify(ctx, token)
+	if err != nil {
+		return mgclients.ClientsPage{}, err
+	}
+
+	cp, err := svc.clients.SearchBasicInfo(ctx, pm)
```

Comment:
@nyagamunene 

Instead of `SearchBasicInfo` can we have `SearchClients` ?

Reply:
We can rename it to `SearchClients` no problem